### ISSUE TITLE
how to stop html-minifier from erasing my tag

### ISSUE THREAD

**Issue by whats1thingnow** [kangax/html-minifier#645]
hello,

was wondering how I can keep a certain fragment of code because it's currently breaking my html after using html-minifier

for example, my index.html has:`<div class="img"> 
        <script type="text/javascript">
            {document.write('<a href="https://link.com/?querystring='+value+'">')}</script>
            <img src="pic01.jpg" style="width: 100%;max-width:500px" border="0">
        </a>         
    </div>`

however, after minify, it becomes:
`<div class="img"> <script>document.write('<a href="https://link.com/?querystring='+value+'">')</script> <img src="pic01.jpg" style="width:100%;max-width:500px" border="0"> </div>`

basically, the `</a>` tag is removed...  which is causing breaking the code...

i tried adding the following (reg expression that matches `</a>`):
`ignoreCustomFragments: [/<\s*\/\s*a\s*.*?>/
            ],`

but it still removes the tag...

please let me know how i can specify html-minifier to keep the tag i want

thanks!

**Comment by alexlamsl** [comment]
Please use [htmlmin:ignore](https://github.com/kangax/html-minifier/blob/gh-pages/README.md#ignoring-chunks-of-markup)

**Comment by alexlamsl** [comment]
Not sure what error is in your case, but `ignoreCustomFragments` also works:

```
minify(`<div class="img"> <script type="text/javascript"> {document.write('<a href="https://link.com/?querystring='+value+'">')}</script> <img src="pic01.jpg" style="width: 100%;max-width:500px" border="0"> </a> </div>`,{ignoreCustomFragments:[/<\s*\/\s*a\s*.*?>/]});
```
